lib/trino-parquet/src/main/java/io/trino/parquet/predicate/TupleDomainParquetPredicate.java

```diff
@@ -214,19 +215,16 @@ public static Domain getDomain(
 
     private static Domain getDomain(
             Type type,
-            Object min,
-            Object max,
+            List<Object> minimums,
+            List<Object> maximums,
```

Comment:
does `minimums.get(i)` and `maximums.get(i)` constitute a range?
or what is the semantics of the params?

Reply:
Yes, exactly. I guess I could build a list of some type of range object, instead... (I mostly recovered the old shape of the code before the latest PR)

Reply:
Fine as is, but consider adding a comment.